modules/cart/src/Plugin/QueueWorker/CartExpire.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Drupal\commerce_cart\Plugin\QueueWorker;
+
+use Drupal\commerce\TimeInterface;
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Removes an expired Cart.
+ *
+ * @QueueWorker(
+ *  id = "cart_expirations",
+ *  title = @Translation("Commerce Cart expiration"),
+ *  cron = {}
+ * )
+ */
+class CartExpire extends QueueWorkerBase implements ContainerFactoryPluginInterface {
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The commerce time service.
+   *
+   * @var \Drupal\commerce\Time
+   */
+  protected $time;
+
+  /**
+   * Constructs a new CartBlock.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin ID for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   * @param \Drupal\commerce\TimeInterface $time
+   *   The time service.
+   */
+  public function __construct(array $configuration,
+                              $plugin_id,
```

Comment:
I know PhpStorm can do this, but regardless length the constructor stays on same line